# Data Science Tools and Ecosystem

In this notebook, Data Science Tools and Ecosystem are summarized.

Some of the popular languages that Data Scientists use are:
1. Python
2. R
3. SQL
4. Java
5. Julia
6. Scala
7. SAS


Some of the commonly used libraries used by Data Scientists include:
1. Pandas
2. Matplotlib
3. NumPy
4. Scikit-learn
5. SciPy
6. Seaborn
7. Beautiful Soup
8. Plotly

| Data Science Tools |
| :--:                 |
| Jupyter Notebooks  |
| RStudio            |
| Apache Hadoop      |
| Apache Spark       |
| TensorFlow         |

### Below are a few examples of evaluating arithmetic expressions in Python

This a simple arithmetic expression to mutiply then add integers

In [1]:
(3*4)+5

17

This will convert 200 minutes to hours by diving by 60

In [5]:
200/60

3.3333333333333335

3.33 hours is 3 hours and 20 minutes

__Objectives:__
- Python
- SQL
- Pandas
- RStudio

## Author
Georgi Zahariev